modules/userdirectory-brightspace/src/main/java/org/opencastproject/userdirectory/brightspace/BrightspaceUserProviderFactory.java

```diff
@@ -152,9 +153,14 @@ public void updated(String pid, Dictionary properties) throws ConfigurationExcep
     logger.debug("creating new brightspace user provider for pid={}", pid);
 
 
-    BrightspaceUserProviderInstance provider = new BrightspaceUserProviderInstance(pid,
-            new BrightspaceClientImpl(urlStr, applicationId, applicationKey, systemUserId, systemUserKey), org, cacheSize,
-            cacheExpiration, adminUserName);
+    BrightspaceUserProviderInstance provider = new BrightspaceUserProviderInstance(
+        pid,
+        new BrightspaceClientImpl(urlStr, applicationId, applicationKey, systemUserId, systemUserKey),
+        org,
+        cacheSize,
+        cacheExpiration,
+        adminUserName
```

Comment:
Okay, I hate to be that person, but this is an absolute pet peeve of mine. :D I think this takes up way too much space now for not that much benefit.